Slims/slims.py

```diff
@@ -0,0 +1,183 @@
+                                  .add(equals('cntn_fk_contentType', 22))
+                                  .add(not_equals('cntn_cstm_runTag', self.run_tag)))
+        # can find the run tags of fastqs from additional runs. can use this to find the fastqs in demultiplexdir or download them from hcp. need to cat fastqs from different runs before starting pipeline. or maybe it works without merging the fastqs first, need to check this...
+        print(more_fastqs)
+        for fq in more_fastqs:
+            print(fq.cntn_cstm_runTag.value)
+
+        return self._fastq
+
+
+# Haven't used the bioinformatics property for anything yet. Might not need it. 
+#    @property
+#    def bioinformatics(self):
+#        if not self.run_tag:
+#            raise Exception('Can not fetch fastq without a set run tag.')
+#
+#        if not self._bioinformatics:
+#            records = slims.fetch('Content', conjunction()
+#                                  .add(equals('cntn_id', self.sample_name))
+#                                  .add(equals('cntn_fk_contentType', 23))
+#                                  .add(equals('cntn_cstm_runTag', self.run_tag)))
+#
+#            if len(records) > 1:
+#                raise Exception('More than 1 bioinformatics somehow.')
+#
+#            if records:
+#                self._bioinformatics = records[0]
+#
+#        return self._bioinformatics
+
+
+def translate_slims_info(record):
+    sample_name = record.cntn_id.value
+
+    # 29 = WOPR
+    # 54 = wgs_somatic
+
+    pipeline_pks = record.cntn_cstm_secondaryAnalysis.value
+    pcr = record.cntn_cstm_pcr.value
+
+    investigator = 'CGG'  # NOTE: Needed?
+
+    department_record = slims.fetch_by_pk('ReferenceDataRecord', record.cntn_cstm_department.value)
+    department = department_record.rdrc_name.value  # Format KK
+    responder_records = [slims.fetch_by_pk('ReferenceDataRecord', pk) for
+                         pk in department_record.rdrc_cstm_responder.value]
+    responder_emails = [rec.rdrc_cstm_email.value for rec in responder_records]
+
+    is_research = record.cntn_cstm_research.value
+    research_project = record.cntn_cstm_researchProject.value
+
+    is_priority = True if record.cntn_cstm_priority.value else False
+
+    gender = record.gender.value
+
+    is_trio = record.cntn_cstm_trio.value
+    trio_id = record.cntn_cstm_trioID.value
+    trio_role = record.cntn_cstm_trioRole.value
+
+    tumorNormalType = record.cntn_cstm_tumorNormalType.value
+    tumorNormalID = record.cntn_cstm_tumorNormalID.value
+
+    tertiary_analysis = record.cntn_cstm_tertiaryAnalysis.value
+
+    master = {
+        'content_id': sample_name,
+        'investigator': investigator,
+        'department': department,
+        'responder_mails': responder_emails,
+        'is_research': is_research,
+        'research_project': research_project,
+        'gender': gender,
+        'is_priority': is_priority,
+        'pcr': pcr,
+        'is_trio': is_trio,
```

Comment:
Is the trio information relevant for this pipeline? If not you might as well remove it and make the code a bit cleaner.